site/src/modules/templates/TemplateCard/TemplateCard.tsx

```diff
@@ -17,7 +17,9 @@ export const TemplateCard: FC<TemplateCardProps> = ({
   ...divProps
 }) => {
   const navigate = useNavigate();
-  const templatePageLink = `/templates/${template.name}`;
+  const organizationId =
+    template.organization_id || "00000000-0000-0000-0000-000000000000";
+  const templatePageLink = `/templates/${template.name}?orgId=${organizationId}`;
```

Comment:
- `template.organization_id` is guaranteed to be well defined here, no need for a fallback.
- we're going to need to add a new route in router.tsx for something like `/templates/:organization/:template`. we don't use `?` for this sort of thing anywhere else in the app, so it'd be weird here.

Reply:
I was split deciding between changing the route or query the query param, I can definitely change the route if you feel that makes more sense. I guess if organization is a fundamental and required part of the information displayed then it makes sense to make it part of the route

Reply:
yeah, it definitely needs to be a routing change.